Client/Frontend/Browser/URLBarView.swift

```diff
@@ -682,27 +682,27 @@ extension URLBarView {
 }
 
 extension URLBarView: Themeable {
-
     func applyTheme() {
         locationView.applyTheme()
         locationTextField?.applyTheme()
         actionButtons.forEach { $0.applyTheme() }
         tabsButton.applyTheme()
 
         progressBar.setGradientColors(startColor: UIColor.theme.loadingBar.start, endColor: UIColor.theme.loadingBar.end)
-        locationBorderColor = UIColor.theme.urlbar.border.withAlphaComponent(0.3)
```

Comment:
G90A10 is the same color as the current one, G50.withAlpha(0.3). I changed this to use the former. This allows the color theme to control the alpha also.